README.md

```diff
@@ -31,6 +31,18 @@
   </a>
 </p>
 
+<p align="center">
+  <a href="https://pennylane.ai/carnival/">
+    <img width=100% src="https://raw.githubusercontent.com/PennyLaneAI/pennylane/master/doc/_static/carnival_splash_bg.png">
+  </a>
+</p>
+
+<p align="center">
+  Announcing <strong>PennyLane Quantum Carnival</strong>. We're throwing a party to celebrate the 3rd anniversary of Pennylane's release! Throughout November, we will have special events, announcements, and releases, all happening under one big tent. <a href="https://pennylane.ai/carnival">Join the fun!</a>
+</p>
+
+<hr>
+

```

Comment:
This PR will remove the ticket that was just added to the README. We'll need to `git rm` this page from this PR and make sure that when the release candidate is merged into master that it takes the readme from `master`

Reply:
Good catch, thank you :+1: I'll double-check that we're not updating the `README.md` file at the time we're merging the release candidate into `master`.